### ISSUE TITLE
Moving multiple users at once

### ISSUE THREAD

**Issue by derlade** [mumble-voip/mumble#3897]
I sometimes find myself in need of moving multiple users to another channel. It would be nice if there was a way to do that other than dragging every single user into another channel. I would suggest Shift or Ctrl functionality, where you could highlight some users first before moving all selected users at once.

**Comment by Krzmbrzl** [comment]
When implementing this there should be a clear approach as to what happens with other actions invoked on a selected user. E.g. what happens if I select multiple users and then trigger an action originally intended for a single user only (e.g. show user status).

I guess there are a few options here:
1) Disable these options altogether as long as more than one user is selected
2) Invoke the action on the user selected first
3) Invoke the action on the user selected last

**Comment by deluxghost** [comment]
> Disable these options altogether as long as more than one user is selected
> Invoke the action on the user selected first
> Invoke the action on the user selected last

as a user, only the first way is acceptable

**Comment by Krzmbrzl** [comment]
I tend to agree with you. Unfortunately I think this is also the option that requires the biggest code-changes when introducing this xD
Nonetheless I think this feature could bring more harm than use, if implemented sloppily

**Comment by Artusamak** [comment]
Huge +1 on this feature request, that would be awesome!

For the selected action, it could apply to all the selected users (eg: mute this person). For the single executable actions as show user info, the last one selected sounds like a good option (since disabling the other actions is too complex).

Thank you for your work <3.
